### ISSUE TITLE
JavaHL binding removed from subversion formula.

### ISSUE THREAD

**Issue by pahjbo** [Homebrew/homebrew-core#122148]
### `brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output

```shell
HOMEBREW_VERSION: 3.6.18-28-g8be5846
ORIGIN: https://github.com/Homebrew/brew
HEAD: 8be5846c517e193d6cf7232083f7882bbafbb62f
Last commit: 3 weeks ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 7d2d8400876d962625cda79aaa8473e9b1c2cd1d
Core tap last commit: 25 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.T479r6oV3t/org.macosforge.xquartz:0
HOMEBREW_MAKE_JOBS: 12
HOMEBREW_NO_INSTALL_CLEANUP: set
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: dodeca-core 64-bit kabylake
Clang: 14.0.0 build 1400
Git: 2.39.0 => /usr/local/bin/git
Curl: 7.79.1 => /usr/bin/curl
macOS: 12.6.3-x86_64
CLT: 14.2.0.0.1.1668646533
Xcode: 14.2


% brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Some installed kegs have no formulae!
This means they were either deleted or installed manually.
You should find replacements for the following formulae:
  python@2
  casacore
  gmp@4
  boost-python
  libodb-boost
  boost@1.55
  szip
  pgplot

Warning: Some installed formulae are deprecated or disabled.
You should find replacements for the following formulae:
  isl@0.18
  mysql@5.6
  py2cairo
  swig@3

Warning: Unbrewed dylibs were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected dylibs:
  /usr/local/lib/libcalc11.dylib

Warning: Unbrewed header files were found in /usr/local/include.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected header files:
  /usr/local/include/ccalc.h
  /usr/local/include/python3.9/greenlet/greenlet.h

Warning: Unbrewed '.la' files were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected '.la' files:
  /usr/local/lib/libvdifio.la

Warning: Unbrewed static libraries were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected static libraries:
  /usr/local/lib/libaddress_sorting.a
  /usr/local/lib/libcalc11.a
  /usr/local/lib/libgpr.a
  /usr/local/lib/libupb.a

Warning: You have unlinked kegs in your Cellar.
Leaving kegs unlinked can lead to build-trouble and cause formulae that depend on
those kegs to fail to run properly once built. Run `brew link` on these:
  pygments
  minikube
  boost
  ipython
  lua@5.1
  handbrake
  pyyaml
  six
  pytorch
  dotty
  wireshark

Warning: Some installed formulae are not readable:
  boost@1.55: Unsupported special dependency :python

  pgplot: Unsupported special dependency :x11

Warning: Some installed formulae are missing dependencies.
You should `brew install` the missing dependencies:
  brew install openssl@3 python@3.8

Run `brew missing` for more details.
```

### Verification

- [X] My "`brew doctor` output" says `Your system is ready to brew.` and am still able to reproduce my issue.
- [X] I ran `brew update` and am still able to reproduce my issue.
- [X] I have resolved all warnings from `brew doctor` and that did not fix my problem.

### What were you trying to do (and why)?

I am one of the people who has been using the javaHL bindings for the last decade. They are use by the subclipse plugin for eclipse.

### What happened (include all command output)?

eclipse complains that it cannot find the javahl library.

### What did you expect to happen?

JavaHL bindings to still be there

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
JavaHL bindings were removed by #113849
```

**Comment by SMillerDev** [comment]
This is a feature request and as such should be provided as a pull request.

**Comment by pahjbo** [comment]
It always feels like a bug when something that was working stops working....

**Comment by SMillerDev** [comment]
That I understand. If you can get anything working (there are some suggestions in #113849) I'd be more than happy to help you get it merged.

**Comment by pahjbo** [comment]
I did read https://github.com/Homebrew/homebrew-core/pull/113849, but I got the impression that it was a potentially irritating maintenance burden having the subversion-javahl formula being mostly the same as the subversion one. I do not know ruby well enough to know if there is something that can be done to inherit/mixin most of the subversion formula to avoid the repetition.

To fix this form me only I have just locally checked out the earlier version of the subversion formula and built from source.
